roles/collectd-client/tasks/main.yml

```diff
@@ -9,6 +9,10 @@
     - collectd
     - collectd-utils
     - collectd-core
+    - lm-sensors
+
+- name: load sensors
+  command: sensors-detect < /dev/null
```

Comment:
command won't work here for shell redirection. You'll need shell. Also if we're using a command family task it'd be nice to specifically define change, so subsequent runs aren't littered with false positive changes.